magento/magento2
# Vague error message for invalid url_key for category

### Description
**Problem**: vague error message for invalid url_key for category
Example error message:
`Invalid URL key`

**Solution**: Use addFailedCategory() to add category to error message
Example error message:
`1. Category " has not been created. Invalid URL key in row(s): 1`

### Fixed Issues (if relevant)

1. magento/magento2#10697: Product Import: Additional data: Invalid URL key

### Manual testing scenarios
1. System > Import > Products
2. Choose "Add/Update
3. Select a CSV file containing a "categories" column with a comma as the first character
4. Click Check Data then Import

### Contribution checklist
